packages/core/.eslintrc.cjs

```diff
@@ -51,7 +51,20 @@ const commonRules = {
   "arrow-parens": "off",
   "lodash/import-scope": [2, "member"],
   "implicit-arrow-linebreak": "off",
-  "import/no-extraneous-dependencies": ["error", { devDependencies: true }]
+  "import/no-extraneous-dependencies": [
+    "error",
+    {
+      devDependencies: [
+        "src/scripts/**/*.ts",
+        "src/storybook/**/*",
+        "src/tests/**/*.ts",
+        "src/utils/jest-utils.js",
+        "**/__tests__/**/*.{js,ts,tsx}",
+        "**/__stories__/**/*.{js,ts,tsx}",
+        "src/components/Combobox/components/ComboboxConstants.ts"
```

Comment:
Why this?

Reply:
Because of what we saw in the issue of ComboboxConstants with preact import
I'll handle it in a different PR to not mix בשר & חלב